templates/pkg/crd_sdk.go.tpl

```diff
@@ -180,6 +185,7 @@ func (rm *resourceManager) sdkCreate(
 		return nil, err
 	}
 {{ $createCode := GoCodeSetCreateOutput .CRD "resp" "ko.Status" 1 }}
+{{ $setCustomOutputOperation := .CRD.GetCustomSetOutputOperation .CRD.Ops.Create }}
```

Comment:
ditto...

Reply:
Moved the code and used `{{ if $setOutputCustomMethodName := ` to combine into one statement.